src/Microsoft.TestPlatform.Common/Telemetry/MetricsCollection.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.Common.Telemetry
+{
+    using System.Collections.Generic;
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+
+    /// <inheritdoc />
+    /// <summary>
+    /// This Class will collect Metrics.
+    /// </summary>
+    public class MetricsCollection : IMetricsCollection
```

Comment:
>MetricsCollection [](start = 17, length = 17)

Do we really need this class. Cant we just do with a simple dictionary instead ?